js/validation.js

```diff
@@ -0,0 +1,80 @@
+const id_input = document.getElementById("id-input");
+// const pw_input = document.getElementById('pw-input');
+const login_btn = document.getElementById("login-button");
+const invalid_id = document.getElementById("invalid-id");
+const invalid_pw = document.getElementById("invalid-pw");
+
+const user_input_email = document.getElementById("user-input-email");
+const email_icon = document.getElementById("email-icon");
+const user_input_pw = document.getElementById("user-input-pw");
+const pw_icon = document.getElementById("pw-icon");
+
+// valid 함수: 전체 유효성 체크
+const valid = () => {
+    if(checkId()){
+        if(checkPw()){
+            return true햐;
+        }
+    }
```

Comment:
true햐는 뭐냐 ㅋㅋ
```suggestion    
    checkId() && checkPw() ? true : false
```

코드를 쓸때 내 코드를 다른 사람이 읽었을때 쉽게 이해 가능한지 고민해보면서 쓰면 좋음
읽기 좋은 코드의 기준중 하나는 간결함이니까 중첩 조건문보다는 삼항연산자가 더 나은대안이 될 수 있음

Reply:
ㅋㅋㅋㅋㅋ저게뭐람..